src/main/java/org/opentripplanner/transit/raptor/api/transit/RaptorTimeTable.java

```diff
@@ -20,6 +22,20 @@
      */
     T getTripSchedule(int index);
 
+    /**
+     * Get the arrival times of all trips at a specific stop index, sorted by time. The returned
+     * value takes the index the trip schedule index in pattern starting at 0 as input and returns
```

Comment:
My brain is having a parse error. I think there are too many words in this sentence.